cmd/elastic-cloud/delete-stale.go

```diff
@@ -35,37 +25,27 @@ type IndexSettings struct {
 
 type Indices map[string]IndexSettings
 
-var setupLog = ctrl.Log.WithName("setup")
-
 func DeleteStaleIndices(c *cli.Context) error {
 	force := c.Bool("force")
-	cloudId := c.String("EC_DEPLOYMENT_CLOUD_ID")
+	apiKey := c.String("deployment-api-key")
```

Comment:
Refactored this to just defined the api key as a hidden flag on the command with envvar overrides. Allows us to ditch `envconfig` package.